user-guide/Reference/Soft-launch_options/BrokerGateway_Migration.md

```diff
@@ -0,0 +1,158 @@
+---
+uid: BrokerGateway_Migration
+---
+
+# Migrating to BrokerGateway
+
+Starting from DataMiner 10.5.0/10.5.2, migrating from the SLNet-managed NATS solution (NAS and NATS services) to the BrokerGateway-managed NATS solution (nats-server service) is possible as a [soft-launch feature](xref:SoftLaunchOptions), if the [BrokerGateway](xref:Overview_of_Soft_Launch_Options#brokergateway) soft-launch option is enabled.
+
+From DataMiner 10.6.0 onwards, this migration will be done automatically during a DataMiner upgrade.
+
+## Prerequisites
+
+Before you start the migration, the entire cluster must have been running smoothly for some time. This means that all DataMiner Agents in the cluster must be online and function together for some time. To confirm that the DMS is ready to be migrated, run the [Verify NATS Migration Prerequisites](xref:BPA_NATS_Migration_Prerequisites) BPA.
```

Comment:
Could we adjust this to be more specific? I think the "for some time" will lead to questions because it's a bit vague.

Reply:
There is a random timer running after startup that will attempt to fix the Single Source of Truth if it wasn't already correct before. The maximum of this timer is 10 minutes, so perhaps we should just say "The entire cluster must have been running for at least 15 minutes".